sdk/search/azure-search-documents/tests/perfstress_tests/search_documents.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure_devtools.perfstress_tests import PerfStressTest
+
+from azure.core.credentials import AzureKeyCredential
+from azure.search.documents import SearchClient as SyncAppConfigClient
+from azure.search.documents.aio import SearchClient as AsyncAppConfigClient
+
+
+class SearchDocumentsTest(PerfStressTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        api_key = self.get_from_env("AZURE_SEARCH_API_KEY")
+        service_endpoint = os.getenv("AZURE_SEARCH_SERVICE_ENDPOINT")
+        index_name = os.getenv("AZURE_SEARCH_INDEX_NAME")
+        key = os.getenv("AZURE_SEARCH_API_KEY")
+        self.service_client = SyncAppConfigClient(service_endpoint, index_name, AzureKeyCredential(api_key))
+        self.async_service_client = AsyncAppConfigClient(service_endpoint, index_name, AzureKeyCredential(api_key))
+
+    async def global_setup(self):
+        await super().global_setup()
```

Comment:
Do we need to pre-load any documents into the index in order for the search to actually return results?
Otherwise the perf test might be skipping some results deserialization logic?
And if the returned results are pagable, then iterating through them would also be an important part. Maybe add a configuable `--num-documents` flag to set up the number of results we would expect to be returned by the search?

Reply:
Yes. This is covered in the pre-configuration. :)

Azure offers some shared data for customers to try the search service.

https://docs.microsoft.com/en-us/azure/search/search-indexer-tutorial

Importing data and creating index are costy and usually user only needs it once so we don't want to run perf tests against them.

Reply:
Yes. This is covered in the pre-configuration. :)

Azure offers some shared data for customers to try the search service.

https://docs.microsoft.com/en-us/azure/search/search-indexer-tutorial

Importing data and creating index are costy and usually user only needs it once so we don't want to run perf tests against them.

Reply:
When you say it's covered in the pre-configuration - there's nothing in the readme about it that I can see?